### ISSUE TITLE
Rearrange Settings Menu

### ISSUE THREAD

**Issue by RageGamerBoi** [vmiklos/plees-tracker#391]
To remove clutter from the corner menu I suggest moving the "Import File", "Import from Calendar", "Export to File", "Export to Calendar", "Delete All Sleep", "Documentation", and "About" sections into the settings menu, and moving the "Dashboard" and "Past Sleeps" sections to their own "Sleep Settings" menu.

It would look something like this (current on the left, proposed on the right):

The corner menu:

![dashboard](https://github.com/vmiklos/plees-tracker/assets/105567407/91de96ad-b72a-4353-82b7-640abab1c898)

The settings menu:

![Settings](https://github.com/vmiklos/plees-tracker/assets/105567407/694a09a4-7ee9-4175-adee-e38a922af74a)

The proposed new sleep settings menu:

![Sleep-Settings](https://github.com/vmiklos/plees-tracker/assets/105567407/8c82a7b1-2df0-437a-9b36-6c3aac81563e)

This is all subject to change, but I like this layout a little better.
It feels more organized and simple.

Thank you.

**Comment by vmiklos** [comment]
Reducing the amount of menu items in the menu would be indeed good. I'm not sure if the settings activity is a good place to host such removed menu items: the settings activity is meant to set values for preferences, not dispatch random user actions, like the menu does.

I wonder if there are apps that do something like this, where we could copy from, as a UX pattern.

**Comment by RageGamerBoi** [comment]
Sure; I took inspiration from OpenTracks, Paisa, Transistor, OBS Blade, Trail Sense, Tracker Control, and Catima
All put the about page, documentation links, etc. in the settings page where they're out of the way.
The ones that have data import/export put it in settings as well because, presumably, the user won't be regularly doing that.

**Comment by aliraeisdanaei** [comment]
I really agree with @LinuxinaBit, and I take on this issue.

**Comment by phpmoli** [comment]
I agree with all of this, except the separate "Sleep settings" menu. I would put all of these into the Settings menu. It can be renamed to "Options", if the problem is Import/Export is not a setting.

You press Import/Export once a year, Delete all sleeps/Documentation/About once every 10 years. These could be absolutely tucked away way-way deep.

**Comment by phpmoli** [comment]
Also, i would put the Import/Export to the top of the list. I know, usually, it is at the bottom, but it is for sure more used, than changing the Dark mode, and stuff.
And if the Import/Export is at the top, the change in interaction from old to new would be:

1. tap kebab menu
2. tap Export

so 2 taps, would change to:

1. if #391 and #606 is done, tap cog icon in the action bar
2. tap Export button which appears at the top of the Settings screen

so, 2 taps.
